README.md

```diff
@@ -2,23 +2,42 @@
 
 [![Build Status](https://travis-ci.org/allegro/ralph-cli.svg?branch=develop)](https://travis-ci.org/allegro/ralph-cli)
 [![Coverage Status](https://coveralls.io/repos/github/allegro/ralph-cli/badge.svg?branch=develop)](https://coveralls.io/github/allegro/ralph-cli?branch=develop)
+[![Documentation Status](https://readthedocs.org/projects/ralph-cli/badge/?version=latest)](http://ralph-cli.readthedocs.io/en/latest/?badge=latest)
 
-A command-line interface for Ralph.
+`ralph-cli` is a command-line interface for [Ralph][ralph]. At this moment, its
```

Comment:
At this moment you can use it for hardware discovery with the `scan` command, but we will extend the functionality in the feature.